### PR TITLE
File added: all changes done in Assignment 5 and 6

### PR DESCRIPTION
**Pull request by dayanandviveki**

### CODE REVIEW

**Review by somesh6797 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+@media only screen and (max-width: 350px) {
+    body {
+        visibility: hidden
+    }
+}
+body .headline {
+    font-size: 60px;
+    text-align: end
+    ;
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: add semicolon in the same line

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    }
+}
+body .headline {
+    font-size: 60px;
+    text-align: end
+    ;  
+    color: whitesmoke;
+}
+body ul {
+    margin-left:4rem;
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: add space after colon always make this change everywhere

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    list-style-type:square;
+    color:green;
+}
+body .links a {
+    text-decoration:none;
+    font-size: 30px;
+    margin: 10px;
+    color: wheat;
+}
+.info,
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: remove this

File: Dayanand Viveki/HTML/Assignment5/biodata.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <title>Learn HTML,CSS extrenal stylesheet</title>
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/biodata.html]: always add space after comma make this change everywhere

File: Dayanand Viveki/HTML/Assignment5/biodata.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                        explicabo a rem quos sit error qui alias quibusdam fugit excepturi, perferendis nesciunt unde, omnis in officiis! Aperiam culpa
+                        similique explicabo quos neque numquam possimus praesentium? Soluta vel nam voluptatibus nihil vero facilis quia molestiae, sit quasi
+                        vitae ipsa sint, adipisci nobis commodi aut! Illo suscipit esse dolorem impedit aut praesentium.
+                    </p>
+                </span>
+                <span class="headline">
+                    <p>More About Me</p>
+                </span>
+                <span class="list-option">
+                    <span class="links">
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/biodata.html]: do not add block level elements inside inline elements

File: Dayanand Viveki/HTML/Assignment5/biodata.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                        dolorem quis ad qui asperiores facere dolore harum maiores labore deserunt eveniet ipsa. Rerum voluptas adipisci velit explicabo sunt
+                        quae et sed! Libero voluptatibus dicta officia iure dolorem. Expedita eaque sapiente magni aliquam eligendi ad perspiciatis adipisci
+                        voluptatum alias maxime quasi numquam facilis cum suscipit, sed fugiat iusto minima neque. Quam architecto, voluptates adipisci
+                        laudantium sit autem earum non a exercitationem veniam dolorum, ullam ratione temporibus eligendi doloribus. Soluta est voluptates
+                        explicabo a rem quos sit error qui alias quibusdam fugit excepturi, perferendis nesciunt unde, omnis in officiis! Aperiam culpa
+                        similique explicabo quos neque numquam possimus praesentium? Soluta vel nam voluptatibus nihil vero facilis quia molestiae, sit quasi
+                        vitae ipsa sint, adipisci nobis commodi aut! Illo suscipit esse dolorem impedit aut praesentium.
+                    </p>
+                </span>
+                <span class="headline">
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/biodata.html:32]: remove span give style to p tag only

File: Dayanand Viveki/HTML/Assignment6/assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,43 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    display: flexbox;
+}
+.font-size-35 {
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment6/assets/css/style.css]: add underscore in class names remove - and make this change everywhere

File: Dayanand Viveki/HTML/Assignment6/personaldetails.html [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+            </div>
+            <hr />
+            <div class="headline font-size-25 italic-font">
+                <p>Personal Details</p>
+            </div>
+            <div class="form-item">
+                <form action="">
+                    <span>
+                        <label for="fname">Frist Name:</label>
+                        <input type="text" id="fname" placeholder="First Name" />
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment6/personaldetails.html]: placeholders second name should start with small letter like First name make this change in all input boxes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

File: Dayanand Viveki/HTML/Assignment5/biodata.html (hunk 6 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/biodata.html:32]: It is easy to give margine both the para through span tag

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css (hunk 2 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: ok

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css (hunk 3 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: ok

File: Dayanand Viveki/HTML/Assignment5/biodata.html (hunk 4 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/biodata.html]: ok mdm

File: Dayanand Viveki/HTML/Assignment5/biodata.html (hunk 5 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/biodata.html]: ok mdm

File: Dayanand Viveki/HTML/Assignment6/assets/css/style.css (hunk 7 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment6/assets/css/style.css]: ok mdm

File: Dayanand Viveki/HTML/Assignment6/personaldetails.html (hunk 8 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment6/personaldetails.html]: ok mdm

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+body ul {
+    margin-left: 4rem;
+    list-style-type: square;
+    color: green;
+}
+body ul li {
+    font-size: 30px;
+    margin-left: 60px;
+}
+.info ,
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: include it in a single line like .info, .links {styles u want to give}

File: Dayanand Viveki/HTML/Assignment5/biodata.html [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <title>Learn HTML, CSS extrenal stylesheet</title>
+    </head>
+    <body>
+        <div class="main_block">
+            <p class="name">Dayanand Viveki</p>
+            <div class="aboutme">
+                <span >
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment5/biodata.html]: remove span

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css (hunk 9 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/assets/css/style.css]: ok

File: Dayanand Viveki/HTML/Assignment5/biodata.html (hunk 10 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/biodata.html]: ok

File: Dayanand Viveki/HTML/Assignment6.2/personalfulldetails.html [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+            <div class="headline font_size_35">
+                <p>MI-11 Inter HTML Assesment</p>
+            </div>
+            <hr />
+            <div class="headline font_size_25 italic_font">
+                <p>Personal Details</p>
+            </div>
+            <div class="form_item">
+                <form action="">
+                    <span>
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment6.2/personalfulldetails.html]: remove all span tags no need of including lable and input tag inside them

File: Dayanand Viveki/HTML/Assignment6/assets/css/style.css [hunk 12]
```diff
@@ -0,0 +1,47 @@
+* {
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment6/assets/css/style.css:1]: check this.file content is duplicate with above css file dont create two css files if not necessary

File: Dayanand Viveki/HTML/Assignment6/personaldetails.html [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+            <div class="headline font_size_35">
+                <p>MI-11 Inter HTML Assesment</p>
+            </div>
+            <hr />
+            <div class="headline font_size_25 italic_font">
+                <p>Personal Details</p>
+            </div>
+            <div class="form_item">
+                <form action="">
+                    <span>
```
Comment (SnehaMakade) [inline Dayanand Viveki/HTML/Assignment6/personaldetails.html:21]: remove all span tags

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do proper changes

File: Dayanand Viveki/HTML/Assignment6.2/personalfulldetails.html (hunk 11 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment6.2/personalfulldetails.html]: ok mdm

**Pull request closed without merging**